app/views/placements/schools/placements/index.html.erb

```diff
@@ -20,7 +20,7 @@
               <% @placements.each do |placement| %>
                 <% body.with_row do |row| %>
                   <% row.with_cell(text: govuk_link_to(
-                    placement.subject_names,
+                    placement.subject_name,
```

Comment:
In fact, maybe we actually _should_ do the refactor in this PR. Since we're already renaming `#subject_names` to `#subject_name`, maybe we should just go ahead and rename it to `#title`.